index.js

```diff
@@ -0,0 +1 @@
+module.exports = { fixturesPath: require('path').join(__dirname, 'fixtures') }
```

Comment:
Exporting an object with names keys as the default/main is a bit of an anti pattern. Why not do:

```suggestion
exports.fixturesPath = require('path').join(__dirname, 'fixtures')
```